Documentation/Blazorise.Docs/Pages/Docs/Extensions/Snackbar/SnackbarPage.razor

```diff
@@ -104,9 +104,12 @@
     <DocsAttributesItem Name="Multiline" Type="bool" Default="false">
         Allow snackbar to show multiple lines of text.
     </DocsAttributesItem>
-    <DocsAttributesItem Name="DefaultInterval" Type="double" Default="5000">
+    <DocsAttributesItem Name="Interval" Type="double" Default="5000">
```

Comment:
Are we changing the naming of an existing **Parameter**?

Reply:
This was wrong in the docs. It was always named `Interval`.